jans-cedarling/cedarling/src/jwt/mod.rs

```diff
@@ -70,9 +70,9 @@ mod error;
 mod http_utils;
 mod key_service;
 mod log_entry;
+mod status_list;
 mod token;
 mod validation;
-mod validator_store;
 
```

Comment:
The status validation implemented but the mod doc was not updated yet.
Consider updating the doc to include details about the status list validation process and configuration options.